.github/workflows/olm_tests.yaml

```diff
@@ -73,6 +73,13 @@ jobs:
           sleep 1
           kubectl create -f https://github.com/operator-framework/operator-lifecycle-manager/releases/download/${OLM_VERSION}/olm.yaml
 
+      - name: Wait for default CatalogSource to start
+        run: |
+          make wait-for-catalog-source
```

Comment:
Do you feel strong about defining the wait logic in the Makefile? I understand it may be to factorize the logic, but it's really a one-liner, and would make the test workflow easier to read.

Also I think `kubectl wait` supports `--for=jsonpath`, so instead of:

```
$ timeout 120 bash -c 'while [[ "$$(kubectl get catalogsource/'$(CATALOG_SOURCE_NAME)' -n '$(CATALOG_SOURCE_NAMESPACE)' -o json | jq -r .status.connectionState.lastObservedState)" != "READY" ]]; do sleep 5 && echo "$$(kubectl get catalogsource/'$(CATALOG_SOURCE_NAME)' -n '$(CATALOG_SOURCE_NAMESPACE)' -o json | jq -r .status.connectionState.lastObservedState)" ; done'
```

This could be:

```
$ kubectl wait -n ${CATALOG_SOURCE_NAMESPACE} catalogsource/'${CATALOG_SOURCE_NAME} --for=jsonpath='{.status.connectionState.lastObservedState}'=READY 
```

Reply:
Good point, will refactor.